README.md

```diff
@@ -73,17 +73,20 @@ variables.
   plugin. For available configuration, please check the
   https://github.com/buildkite-plugins/docker-buildkite-plugin.
 - `TESTS_TO_SKIP`: specifies a list of tests to be skipped.
+- `TIMEOUTS`: overrides the timeout value for specific tests
 
 The variable `TESTS_TO_SKIP` is specified as a JSON list with the names
-of the tests to be skipped. The other variables are specified as dictionaries,
-where the first key is `tests` and its value is a list of test names where the
-configuration should be applied; the second key is `cfg` and its value is a
-dictionary with the actual configuration.
-
-For example, we can skip the test `commit-format` and extend the docker plugin 
-specification as follows:
+of the tests to be skipped. The variable `TIMEOUTS` is a dictionary where each
+key is the name of a test and each value is the number of minutes for the
+timeout. The other variables are specified as dictionaries, where the first key
+is `tests` and its value is a list of test names where the configuration should
+be applied; the second key is `cfg` and its value is a dictionary with the
+actual configuration.
+
+For example, we can skip the test `commit-format`, have a timeout of 30 minutes
+for the test `style` and extend the docker plugin specification as follows:
```

Comment:
The example is not using the `style` test in TIMEOUTS.

Reply:
Changed it, thanks!